apps/block_scout_web/lib/block_scout_web/schema/types.ex

```diff
@@ -194,8 +202,10 @@ defmodule BlockScoutWeb.Schema.Types do
     field(:num_registered_validators, :integer)
     field(:min_electable_validators, :integer)
     field(:max_electable_validators, :integer)
-    field(:gold_token, :address_hash)
-    field(:stable_token, :address_hash)
+    field(:gold_token, :address_hash, deprecate: "Use celoToken instead.")
```

Comment:
this builds the graphql schema right? 

isnt part of the GraphQL spec that nothing is ever deprecated?

Reply:
Not really, you can deprecate fields. In our case I think it will even keep working and would display a warning message in the UI.